string first_string
string second_string
---
string changed_string